Project-Euler/Problem019.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Problem 19 - Counting Sundays
+ *
+ * @see {@link https://projecteuler.net/problem=19}
+ *
+ * You are given the following information, but you may prefer to do some research for yourself.
+ * 1 Jan 1900 was a Monday.
+ * Thirty days has September,
+ * April, June and November.
+ * All the rest have thirty-one,
+ * Saving February alone,
+ * Which has twenty-eight, rain or shine.
+ * And on leap years, twenty-nine.
+ * A leap year occurs on any year evenly divisible by 4, but not on a century unless it is divisible by 400.
+ * How many Sundays fell on the first of the month during the twentieth century (1 Jan 1901 to 31 Dec 2000)?
+ *
+ * @author ddaniel27
+ */
+
+// Check if a year is a leap year
+// A leap year occurs on any year evenly divisible by 4, but not on a century unless it is divisible by 400.
+function isLeapYear(year) {
```

Comment:
This should be imported from `Maths/LeapYear.js`.

Reply:
@appgurueu solved!